templates/zerver/help/make-an-announcement.md

```diff
@@ -15,15 +15,15 @@ Announcements on Zulip are sent using the `#announce` stream.
 4. Enter the topic for your announcement in the **Topic** field.
 
     !!! tip ""
-        Your topic name can be no longer than 52 characters.
+        Your topic name cannot be longer than 52 characters.
 
-5. Enter your announcement in the box labeled
- **Compose your message here...** or messaging box.
+5. Enter your announcement in **compose box ** (also known as the messaging box).
 
-6. Finally, you can now send your announcement by clicking the **Send** button.
+6. Finally, you can now send your announcement by clicking the **Send** button or
+  by pressing the enter key if you have checked the button **Press Enter to send**.
```

Comment:
I'd put the `if you have checked..` in brackets. Also, it's not a button, it's a `**Press Enter to send** option`.